client/js/components/GoogleMapSearch/GoogleMapsSearch.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from "react";
+import { View } from "react-native";
+import { GooglePlacesAutocomplete } from "react-native-google-places-autocomplete";
+import styles from "./styles";
+import globalStyles from "../../config/globalStyles";
+
+class GooglePlacesInput extends Component {
+  constructor(props) {
+    super(props);
+  }
```

Comment:
Please remove constructor if you do not have a state and convert this component into a stateless component.